From 8dab6f991d99c886d6dcd1f3c73451d5986f6882 Mon Sep 17 00:00:00 2001
From: Marc Mutz <marc.mutz@qt.io>
Date: Tue, 30 Sep 2025 11:58:51 +0200
Subject: [PATCH 249/553] QWinRegistryKey: remove unused qpair.h include

Amends 87361727ffafbee460970cdb8e72056653fd2443, which removed the
last use of QPair from the file, but not picking to older branches in
case users rely on the transitive include.

Task-number: QTBUG-115841
Change-Id: Idba1980b3309635f24d830e2a922beaeff600c55
Reviewed-by: Friedemann Kleint <Friedemann.Kleint@qt.io>
(cherry picked from commit d856dc2c00a2e89f268550bf87898c122e8d0b23)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/corelib/kernel/qwinregistry_p.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/corelib/kernel/qwinregistry_p.h b/src/corelib/kernel/qwinregistry_p.h
index a1a84e8bd24..bb0c58f84a3 100644
--- a/src/corelib/kernel/qwinregistry_p.h
+++ b/src/corelib/kernel/qwinregistry_p.h
@@ -15,7 +15,6 @@
 // We mean it.
 //
 
-#include <QtCore/qpair.h>
 #include <QtCore/qstring.h>
 #include <QtCore/qstringview.h>
 #include <QtCore/qt_windows.h>
-- 
2.51.2

